### PR TITLE
[11.0] [MIG] hr_employee_age: Migrated the module in v11.

### PR DESCRIPTION
**Pull request by Chandresh-SerpentCS**
- Added run bot link.
- Removed read only attribute from compute field.

### CODE REVIEW

File: hr_employee_age/models/hr_employee.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+    age = fields.Integer(
+        string='Age',
+        compute='_compute_age'
+    )
+
+    @api.multi
+    @api.depends('birthday')
+    def _compute_age(self):
+        for record in self:
```
Comment (JagdishPanchalSCS) [inline hr_employee_age/models/hr_employee.py:20]: you can check validation here can not allow to birthday field Greater then today

**Review by JagdishPanchalSCS — CHANGES_REQUESTED** [review]

File: hr_employee_age/models/hr_employee.py (hunk 1 above)
Comment (Chandresh-SerpentCS) [reply hr_employee_age/models/hr_employee.py:20]: Agreed.

File: hr_employee_age/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2015 Salton Massally (<smassally@idtlabs.sl>).
```
Comment (alexey-pelykh) [inline hr_employee_age/__manifest__.py]: It should be w/o `(C)` as per https://github.com/OCA/maintainer-tools/blob/1ae9b316377defbde06b67e08763f574260ee153/template/module/__manifest__.py#L2

File: hr_employee_age/models/__init__.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# Copyright (C) 2015 Salton Massally (<smassally@idtlabs.sl>).
```
Comment (alexey-pelykh) [inline hr_employee_age/models/__init__.py]: `__init__` fields should go w/o Copyright notice as per https://github.com/OCA/maintainer-tools/blob/master/template/module/models/__init__.py

File: hr_employee_age/views/hr_employee.xml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<?xml version = "1.0" encoding="utf-8"?>
+<odoo>
+    <record id="hr_employee_view_form" model="ir.ui.view">
+        <field name="name">hr.employee.view.form</field>
+        <field name="model">hr.employee</field>
+        <field name="inherit_id" ref="hr.view_employee_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='birthday']" position="after">
```
Comment (alexey-pelykh) [inline hr_employee_age/views/hr_employee.xml]: You may just use `field` instead of `xpath` here

**Review by alexey-pelykh — CHANGES_REQUESTED** [review]

File: hr_employee_age/models/hr_employee.py [hunk 5]
```diff
@@ -27,7 +10,6 @@ class HrEmployee(models.Model):
 
     age = fields.Integer(
         string='Age',
-        readonly=True,
```
Comment (alexey-pelykh) [inline hr_employee_age/models/hr_employee.py]: Why you've decided to remove the `readonly` flag? It's age a definite function of birthday?

**Review by alexey-pelykh — CHANGES_REQUESTED** [review]

**Comment by pedrobaeza** [comment]
Please squash together adjacent "OCA Transbot.." commits using https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

File: hr_employee_age/models/hr_employee.py (hunk 5 above)
Comment (Chandresh-SerpentCS) [reply hr_employee_age/models/hr_employee.py]: @alexey-pelykh, Compute field will be by default read-only, so no need to define readonly=True inside the field definition. I have tested after the remove this attribute. It's working well.

File: hr_employee_age/README.rst [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+shown above.
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/116/11.0
+   
+Known issues / Roadmap
```
Comment (nikul-serpentcs) [inline hr_employee_age/README.rst:34]: @Chandresh-SerpentCS If not need then remove it!

File: hr_employee_age/__manifest__.py [hunk 7]
```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2015 Salton Massally.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Employee Age",
+    'version': '11.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': "Salton Massally <smassally@idtlabs.sl>, "
+            "Odoo Community Association (OCA)",
+    "website": "http://idtlabs.sl",
```
Comment (nikul-serpentcs) [inline hr_employee_age/__manifest__.py:10]: @Chandresh-SerpentCS Replace URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)

File: hr_employee_age/models/hr_employee.py [hunk 8]
```diff
@@ -0,0 +1,30 @@
+# Copyright (C) 2015 Salton Massally.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+from dateutil.relativedelta import relativedelta
+from odoo.exceptions import Warning
```
Comment (nikul-serpentcs) [inline hr_employee_age/models/hr_employee.py:6]: remove `Warning`, here use `USerError`

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Improve Code

**Comment by emagdalenaC2i** [comment]
@pedrobaeza There is other PR for the migration of this module in #458

**Pull request closed without merging**